onecodex/dataframes.py

```diff
@@ -172,7 +172,7 @@ def __init__(self, pandas_obj):
         self._metric = pandas_obj.ocx_metric
         self._rank = pandas_obj.ocx_rank
         self._results = pandas_obj
-        self._all_nan_classification_ids = pandas_obj.ocx_all_nan_classification_ids
+        self.ocx_all_nan_classification_ids = pandas_obj.ocx_all_nan_classification_ids
```

Comment:
the other attributes on this object aren't prefixed with `ocx_`, is there a reason this one is? the previous name `_all_nan_classification_ids` made sense to me, and it's a "private" name too

Reply:
It can't be the same name as the property on `AnalysisMixin` - I can put an underscore though